design/ceph/object/store.md

```diff
@@ -221,6 +221,293 @@ spec:
+
+    #  [EXISTING,OPTIONAL] - data pool. Will be used by default to store bucket objects
+    # default poolPlacement.dataPoolName will override this value, so it is not makes sense to use together with poolPlacements.
+    dataPoolName: rgw-data-pool # results to rgw-data-pool:<CephObjectStore.name>.data
+
+    # New settings:
+    # [OPTIONAL] - defines pool placement targets.
+    # Placement targets control which Pools are associated with a particular bucket.
+    # See: https://docs.ceph.com/en/latest/radosgw/placement/#placement-targets
+    poolPlacements:
+      # Default pool placement:
+      - name: "default"                         # [REQUIRED] - placement id. Placement with name "default" will be used as default,
+                                                # and therefore will override sharedPools.metadataPoolName and sharedPools.dataPoolName if presented.
+                                                # This means that defaul poolPlacement is also mutually exclusive with spec.metadataPool and spec.dataPool
+        metadataPoolName: rgw-meta-pool         # [REQUIRED] results to rgw-meta-pool:<CephObjectStore.name>.index
+        dataPoolName: rgw-data-pool             # [REQUIRED] results to rgw-data-pool:<CephObjectStore.name>.data
+        dataNonECPoolName: ""                   # [OPTIONAL] results to rgw-data-pool:<CephObjectStore.name>.data.nonec
+        # [OPTIONAL] - Storage classes for cold placement target.
+        storageClasses:
+          - name: GLACIER                       # [REQUIRED] StorageClass name. Can be arbitrary but it is better to use one of AWS x-amz-storage-class names.
+            dataPoolName: glacier-data-pool     # [REQUIRED] results to glacier-data-pool:<CephObjectStore.name>.glacier
+      # Pool placement target named "cold"
+      - name: "cold"                            # [REQUIRED] - placement id. Can be arbitrary.
+        metadataPoolName: rgw-cold-meta-pool    # [REQUIRED] results to rgw-cold-meta-pool:<CephObjectStore.name>.cold.index
+        dataPoolName: rgw-cold-data-pool        # [REQUIRED] results to rgw-cold-data-pool:<CephObjectStore.name>.cold.data
+        dataNonECPoolName: ""                   # [OPTIONAL] results to rgw-cold-data-pool:<CephObjectStore.name>.cold.data.nonec
+        # [OPTIONAL] - Storage classes for cold placement target.
+        storageClasses:
+          - name: GLACIER                       # [REQUIRED] StorageClass name. Can be arbitrary but it is better to use one of AWS x-amz-storage-class names.
+            dataPoolName: glacier-data-pool     # [REQUIRED] results to glacier-data-pool:<CephObjectStore.name>.glacier
+      # Pool placement target named "hot"
+      - name: "hot"
+        # example: different placements can share index pool
+        metadataPoolName: rgw-cold-meta-pool    # [REQUIRED] results to rgw-cold-meta-pool:<CephObjectStore.name>.hot.index
```

Comment:
```suggestion
        metadataPoolName: rgw-hot-meta-pool    # [REQUIRED] results to rgw-hot-meta-pool:<CephObjectStore.name>.hot.index
```

Reply:
i wanted to demonstrate that user can use the same pool and we will handle it with rados namespaces. see [comment above](https://github.com/rook/rook/pull/14455/files#diff-cf542a1a819df09579569b1ca4896a8c8530c9437a1ee8f361063b505767aa3dR358).

Reply:
Mixing `hot` and `cold` examples is confusing to me. To keep the common case simple, I'd suggest just using separate pools. Then you could either provide a separate example, or just a sentence that describes that it's acceptable to use a common metadata pool.